infra/blueprint-test/pkg/tft/terraform.go

```diff
@@ -36,6 +36,13 @@ import (
 
 const setupKeyOutputName = "sa_key"
 
+var (
+	commonRetryableErrors = map[string]string{
+		".*FOLDER_TO_DELETE_NON_EMPTY_VIOLATION.*": "Failed to delete non empty folder.",
+		".*SERVICE_DISABLED.*":                     "Required API not enable.",
```

Comment:
Lets add a comment here that API activation is eventually consistent and that even the project_service resource is reconciled there maybe an activation error.
```suggestion
		".*SERVICE_DISABLED.*":                     "Required API not enabled.",
```

Reply:
fixed